go.mod

```diff
@@ -1,38 +1,61 @@
 module github.com/openshift-online/ocm-common
 
-go 1.19
+go 1.21
+
+toolchain go1.21.6
 
 require (
-	github.com/aws/aws-sdk-go-v2 v1.22.2
-	github.com/aws/aws-sdk-go-v2/service/iam v1.27.1
+	github.com/aws/aws-sdk-go v1.51.2
```

Comment:
why would you need this import if we are supposed to use only v2 ?

Reply:
Looks like some of the newly added files from the [doc](https://docs.google.com/document/d/13QxUj510QhqgYKhJGXwjoCFLCURZfq99I0cnBC2NqWc/edit) you directed me to has some references. I will clean this up and push a new commit.